docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>Cool. I can better understand this figure. I think what you are trying to say is that the discord discovered by matrix profile is not what we are looking for. I suggest to do the following instead:</p><p> </p><p>1 - compute the discord index and its nearest neighbor using matrix profile.</p><pre class="ql-syntax" spellcheck="false">idx = np.argmax(MP_PP[:,0])  # start index of discord
nn_idx = MP_PP[:,1]  # start index of nearest neighbor of discord
</pre><p> </p><p> </p><p>2- show two figures only,  and both of them should be <code>T(+)</code>. In one, you plot <code>T(+)</code>   and show the discord and its nearest neighbor subsequences, according to the computed <code>idx</code>  and<code>nn_idx</code>. In another figure, you plot <code>T(+)</code> again but you show  the desirbale subsequences instead. </p><p> </p><p>Please choose a proper title for each figure. For the first figure, we may say: "the discord and its nearest neighbor discovered by the matrix profile", and for the second figure, we may say: "the desirbale subsequences of our interest"</p><p> </p><p>Now, instead of four figures, we only show two figrues, and I think it should be easily understandable. Note that the figure of matrix profile itself (the last two figures shown above) does not help that much. The third figure only shows where the matrix profile is maximum . and the fourth figure might be confusing. So, I think using two lines of code is better than showing two more figures here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>